lib/index.js

```diff
@@ -58,20 +57,31 @@ Vero.prototype.group = request('/users/edit', 'put');
  * @api public
  */
 
-Vero.prototype.track = function(payload, fn) {
+Vero.prototype.track = function(track, fn) {
+  var payload = mapper.track.call(this, track);
```

Comment:
figured this was easier than modifying all of the mapper functions but open to changing if you think it's confusing